extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -319,6 +331,22 @@ export class AmpSelector extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Handles selectUp events.
+   * @param {number} delta
+   */
+  select_(delta) {
+    // Change the selection to the next element in the specified direction.
+    // The selection should loop around if the user attempts to go one
+    // past the beginning or end.
+    const previousIndex = this.options_.indexOf(this.selectedOptions_[0]);
```

Comment:
Will this still work if the user calls `myAmpSelector.selectDown(-1000)`?

Reply:
TIL: Javascript % is not true modulo. 

Adapted the answer here to work: https://stackoverflow.com/a/47354356

I have also added tests to check select{Up,Down}(1001)

Reply:
Nice! This could be a good utility method to add to `src/math.js` since I'm sure it'd be generally useful.

Reply:
Done. Also added tests to `test-math.js` which helped me catch another issue in the code.